### DISCUSSION TITLE
histogram

### DISCUSSION THREAD

**Discussion by dinaf39** [teamalgoritma/community#665] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/130343206/ff01b1cc-9c28-4af5-b1af-8918913ceee3)
memiliki kendala  unsupported operand type(s) for /: 'str' and 'int'

**Comment by nadyadtm** [comment, marked as answer]
Dear Bu @dinaf39 

Menyambung dengan pertanyaan ibu di bagian scatter plot, untuk bagian Xnya perlu menggunakan kolom numerik. Jadi silakan untuk menggunakan kolom numerik yang sesuai dengan tasknya. Terima kasih dan semoga membantu.

**Comment by nadyadtm** [comment]
“Halo @dinaf39, untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah lebih dari 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.”
